Islandora/documentation
# Iiif docs

**GitHub Issue**: Maybe  https://github.com/Islandora/documentation/issues/1184 ?

# What does this Pull Request do?

Adds a page to the User Documentation describing how IIIF is implemented in Islandora 8. View it live at https://github.com/mjordan/documentation/blob/iiif_docs/docs/user-documentation/iiif.md.

# What's new?

Adds a single new page (`docs/user-documentation/iiif.md`) and one image (`docs/assets/osd_collection_mode.png`).

# How should this be tested?

Same as other documentation: read, suggest, correct, etc.

Question: how much of this should be documented as "Paged content"? @seth-shaw-unlv's https://github.com/Islandora/documentation/pull/1333 is currently under review so we should make sure there is no overlap, or that the two pages link to each other where appropriate.

# Additional Notes:

Currently, what is described in this page does not work. See https://github.com/Islandora/documentation/issues/1294 for more info.

# Interested parties
@cderven  @Islandora/8-x-committers
